Wrappers/Python/ccpi/viewer/CILViewer.py

```diff
@@ -632,9 +538,6 @@ def addActor(self, actor):
         '''Adds an actor to the render'''
         return self.showActor(0, actor)
 
-    def startRenderLoop(self):
-        self.iren.Start()
-
     def setInput3DData(self, imageData):
         self.img3D = imageData
```

Comment:
Also this one could be in the base class?

Reply:
It has different functionality in the 2 viewers as it involves resetting of clipping etc. in the 3D viewer

Reply:
Does it here? I think the method could at least be `NotImplemented` in the base class.

Reply:
It is